sdcm/tester.py

```diff
@@ -839,6 +826,105 @@ def get_cluster_baremetal(self):
         params['private_ips'] = self.params.get('monitor_nodes_private_ip')
         self.monitors = cluster_baremetal.MonitorSetPhysical(**params)
 
+    def get_cluster_k8s_gce_minikube(self):
+        self.credentials.append(UserRemoteCredentials(key_file=self.params.get('user_credentials_path')))
+
+        services = get_gce_services(self.params.get('gce_datacenter').split())
+        assert len(services) == 1, "Doesn't support multi DC setup for `k8s-gce-minikube' backend"
+
+        services, gce_datacenter = list(services.values()), list(services.keys())
+
+        self.k8s_cluster = \
+            cluster_k8s.GceMinikubeCluster(minikube_version=self.params.get("minikube_version"),
+                                           gce_image=self.params.get("gce_image_minikube"),
+                                           gce_image_type=self.params.get("gce_root_disk_type_minikube"),
+                                           gce_image_size=self.params.get("gce_root_disk_size_minikube"),
+                                           gce_network=self.params.get("gce_network"),
+                                           services=services,
+                                           credentials=self.credentials,
+                                           gce_instance_type=self.params.get("gce_instance_type_minikube"),
+                                           gce_image_username=self.params.get("gce_image_username"),
+                                           user_prefix=self.params.get("user_prefix"),
+                                           params=self.params,
+                                           gce_datacenter=gce_datacenter)
+
+        self.k8s_cluster.wait_for_init()
+
+        # This should remove some of the unpredictability of pods startup time.
+        scylla_docker_image = f'scylladb/scylla:{self.params.get("scylla_version")}'
```

Comment:
why the whole code installing the operator isn't part of the `ScyllaPodCluster` or `GceMinikubeCluster` ?

Reply:
There are plans to ad more k8s-* backends and it's a place for the code which connects `k8s_cluster` and `db_cluster` parts, as we do for loaders and monitors. We'll see how much of this code will become generic and we'll move it to separate abstraction (maybe.)